src/crypto/key_validator/key_validator_impl.cpp

```diff
@@ -167,4 +166,16 @@ namespace libp2p::crypto::validator {
     return outcome::success();
   }
 
+  outcome::result<void> KeyValidatorImpl::validateEcdsa(
+      const PrivateKey &key) const {
+    // TODO(xDimon): Check if it possible to validate ECDSA key by some way.
```

Comment:
I suggest filing a task or an issue for it and put an ID here.

Unconditional returning of success without any notice about implementation absence looks like a potential pitfall for any further development or execution.